tests/contrib/algoliasearch/test.py

```diff
@@ -144,7 +140,23 @@ def test_patch_all_auto_enable(self):
 
         unpatch()
 
-        self.perform_search('test search')
+        self.perform_search("test search")
 
         spans = self.get_spans()
         assert not spans, spans
+
+    @BaseTracerTestCase.run_in_subprocess(env_overrides=dict(DD_SERVICE="mysvc"))
+    def test_user_specified_service(self):
```

Comment:
I'm guessing the other tests basically cover the default case where `DD_SERVICE` or the global service is otherwise not configured?

Reply:
exactly!